lib/ProductOpener/Redis.pm

```diff
@@ -0,0 +1,43 @@
+use Redis::Client;
+
+BEGIN
+{
+	use vars       qw(@ISA @EXPORT_OK %EXPORT_TAGS);
+	@EXPORT_OK = qw(
+		&push_to_search_service
+
+		);    # symbols to export on request
+	%EXPORT_TAGS = (all => [@EXPORT_OK]);
+}
```

Comment:
Please excuse the drive-by review. I was just doing some hacktoberfest browsing. 😄 
I'm not too familiar with this project, but I think the beginning of the package can be shortened to something like:

```perl
package ProductOpener::Redis;

use strict;
use warnings;

use Exporter qw( import );
use Redis::Client ();

our @EXPORT_OK = qw( push_to_search_service );
our %EXPORT_TAGS = ( all => [@EXPORT_OK] );
```

Reply:
@oalders advice are always welcome ! But for now it's better to be consistent with the rest of the code. If you could write an issue to describe how we should do instead, so that we apply it to all modules at once, it would be great :-) We are kin to modernize our code base one step at a time.

Reply:
@alexgarel sure. I'll put something together and people can decide if there's anything there worth exploring. I had seen a talk on your site given at The Perl Conference last year, so I had been meaning to have a closer look.